pages/vi/vi-github-and-repositories.md

```diff
@@ -34,17 +34,15 @@ Then, turn to your command prompt and type your repository URL in the form of `g
 
 ##### Clone with HTTPS or Use SSH?
 
-Both HTTPS and SSH URLs identify the same remote repositories but use different protocols to access the codebase. Besides HTTPS, which we talked about above, you can also use SSH to do the same thing. You can explore the differences in thisarticle: [connecting-to-github-with-ssh](https://help.github.com/articles/connecting-to-github-with-ssh/).
```

Comment:
I believe this is the correct text except a space is needed between "this" and "article"